services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+) -> QuerySet:
+    movie = Movie.objects.all()
+    if genres_ids:
+        movie = movie.filter(genres__in=genres_ids)
+    if actors_ids:
+        movie = movie.filter(actors__in=actors_ids)
+    return movie
```

Comment:
When filtering movies by genres and actors, consider using `distinct()` at the end of the query to ensure that the result set contains unique movies. This is important because using `filter` with `__in` can result in duplicate entries if a movie matches multiple criteria.